sdk/storage/azure-storage-blob-nio/README.md

```diff
@@ -1,5 +1,6 @@
-# Azure Storage Java NIO Blob plugin library for Java
-## This README is not yet updated for this project and is a skeleton copied from blobs
+# Azure Storage Blob NIO FileSystemProvider
+
+This package allows you to interact with Azure Blob Storage through the standard Java NIO Filesystem APIs.
 
 [Source code][source] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs] | [Samples][samples]
```

Comment:
samples link goes to 404

Reply:
I'm debating whether I should add samples or if it's redundant because once you load the file system provider, it's the same thing as just using normal file apis. Thoughts? @alzimmermsft @kasobol-msft

Reply:
Does NIO have some samples we could link to?

Reply:
I think probably the oracle tutorials are sufficient. https://docs.oracle.com/javase/tutorial/essential/io/fileio.html